contract_compassion/wizard/end_contract_wizard.py

```diff
@@ -32,8 +32,9 @@ class EndContractWizard(models.TransientModel):
     do_transfer = fields.Boolean('I want to transfer the child')
     transfer_country_id = fields.Many2one(
         'compassion.global.partner', 'Country')
+    has_new_hold = fields.Boolean(default=True)

```

Comment:
`has_new_hold = fields.Boolean('Keep child on hold', default=fields.Datetime.now())`
in order to give a more comprehensible name in the view and to set by default to the current time.